### ISSUE TITLE
Not possible to disable automatic update check

### ISSUE THREAD

**Issue by gnugnug** [geluk/pass-winmenu#70]
When I set _check-for-updates_ to _false_, pass-winmenu displays the following popup when starting and exits afterwards, so it's not possible to use the program with the automatic update check disabled:

```
An error occured:
pass-winmenu failed to start (DependencyResolutionException: An exception was thrown while activating 
PassWinmenu.Actions.ActionDispatcher -> 
λ:PassWinmenu.Actions.IAction[] -> 
PassWinmenu.Actions.CheckForUpdatesAction -> 
λ:PassWinmenu.UpdateChecking.UpdateChecker.)
```

**Comment by geluk** [comment]
You're right, seems like this is a regression introduced in the 1.9 release. I've created a fix for this that I'll include in the next version.

**Comment by geluk** [comment]
Fixed in [v1.11](https://github.com/geluk/pass-winmenu/releases/tag/v1.11). If you still encounter any issues, let me know!
